tensorflow/python/kernel_tests/conv_ops_3d_test.py

```diff
@@ -56,11 +69,11 @@ def _SetupValuesForDevice(self, tensor_in_sizes, filter_in_sizes, stride,
 
     # Initializes the input tensor with array containing incrementing
```

Comment:
Can you update this comment? Something like "Initializes the input tensor with array containing numbers from 0 to 1. We keep the input tensor values fairly small to avoid overflowing a float16 tensor during the conv3d."

Reply:
Done.